src/api.js

```diff
@@ -525,83 +472,43 @@ const serviceExtensions = {
   services: async () => {
     const url = new URL(routes.serviceExtensions.base)
 
-    const requestId = 'serviceExtensionsServices'
-    if (serveFromCache(requestId)) {
-      return serveFromCache(requestId)
-    }
-
     const resp = await withAuth(url)
-
     if (!resp.ok) throw Error(resp.statusText)
 
-    const result = await resp.json()
-    updateCache(requestId, result)
-
-    return result?.sort()
```

Comment:
Please keep the sort order of the results, although it might be a better choice to put it into the using code.